Ext.data.JsonP.java_lang_reflect_InvocationTargetException({"tagname":"class","name":"java.lang.reflect.InvocationTargetException","autodetected":{},"files":[{"filename":"InvocationTargetException.js","href":"InvocationTargetException.html#java-lang-reflect-InvocationTargetException"}],"extends":"java.lang.ReflectiveOperationException","members":[{"name":"getCause","tagname":"method","owner":"java.lang.reflect.InvocationTargetException","id":"method-getCause","meta":{}},{"name":"getTargetException","tagname":"method","owner":"java.lang.reflect.InvocationTargetException","id":"method-getTargetException","meta":{}}],"alternateClassNames":[],"aliases":{},"id":"class-java.lang.reflect.InvocationTargetException","short_doc":"InvocationTargetException is a checked exception that wraps\n an exception thrown by an invoked method or constructor. ...","classIcon":"icon-class","superclasses":["java.lang.ReflectiveOperationException"],"subclasses":[],"mixedInto":[],"mixins":[],"parentMixins":[],"requires":[],"uses":[],"html":"<div><pre class=\"hierarchy\"><h4>Hierarchy</h4><div class='subclass first-child'>java.lang.ReflectiveOperationException<div class='subclass '><strong>java.lang.reflect.InvocationTargetException</strong></div></div><h4>Files</h4><div class='dependency'><a href='source/InvocationTargetException.html#java-lang-reflect-InvocationTargetException' target='_blank'>InvocationTargetException.js</a></div></pre><div class='doc-contents'><p>InvocationTargetException is a checked exception that wraps\n an exception thrown by an invoked method or constructor.</p>\n\n<p> <p>As of release 1.4, this exception has been retrofitted to conform to\n the general purpose exception-chaining mechanism.  The \"target exception\"\n that is provided at construction time and accessed via the\n <a href=\"#!/api/java.lang.reflect.InvocationTargetException-method-getTargetException\" rel=\"java.lang.reflect.InvocationTargetException-method-getTargetException\" class=\"docClass\">getTargetException</a>() method is now known as the <i>cause</i>,\n and may be accessed via the Throwable.getCause() method,\n as well as the aforementioned \"legacy method.\"</p>\n\n<p> @see Method\n @see Constructor</p>\n</div><div class='members'><div class='members-section'><div class='definedBy'>Defined By</div><h3 class='members-title icon-method'>Methods</h3><div class='subsection'><div id='method-getCause' class='member first-child not-inherited'><a href='#' class='side expandable'><span>&nbsp;</span></a><div class='title'><div class='meta'><span class='defined-in' rel='java.lang.reflect.InvocationTargetException'>java.lang.reflect.InvocationTargetException</span><br/><a href='source/InvocationTargetException.html#java-lang-reflect-InvocationTargetException-method-getCause' target='_blank' class='view-source'>view source</a></div><a href='#!/api/java.lang.reflect.InvocationTargetException-method-getCause' class='name expandable'>getCause</a>( <span class='pre'></span> ) : Object {java.lang.Throwable}<span class=\"signature\"></span></div><div class='description'><div class='short'>Returns the cause of this exception (the thrown target exception,\n which may be {@code null}). ...</div><div class='long'><p>Returns the cause of this exception (the thrown target exception,\n which may be {@code null}).</p>\n        <p>Available since: <b>1.4</b></p>\n<h3 class='pa'>Returns</h3><ul><li><span class='pre'>Object {java.lang.Throwable}</span><div class='sub-desc'><p>the cause of this exception.</p>\n</div></li></ul></div></div></div><div id='method-getTargetException' class='member  not-inherited'><a href='#' class='side expandable'><span>&nbsp;</span></a><div class='title'><div class='meta'><span class='defined-in' rel='java.lang.reflect.InvocationTargetException'>java.lang.reflect.InvocationTargetException</span><br/><a href='source/InvocationTargetException.html#java-lang-reflect-InvocationTargetException-method-getTargetException' target='_blank' class='view-source'>view source</a></div><a href='#!/api/java.lang.reflect.InvocationTargetException-method-getTargetException' class='name expandable'>getTargetException</a>( <span class='pre'></span> ) : Object {java.lang.Throwable}<span class=\"signature\"></span></div><div class='description'><div class='short'>Get the thrown target exception. ...</div><div class='long'><p>Get the thrown target exception.</p>\n\n<p> <p>This method predates the general-purpose exception chaining facility.\n The Throwable.getCause() method is now the preferred means of\n obtaining this information.</p>\n<h3 class='pa'>Returns</h3><ul><li><span class='pre'>Object {java.lang.Throwable}</span><div class='sub-desc'><p>the thrown target exception (cause of this exception).</p>\n</div></li></ul></div></div></div></div></div></div></div>","meta":{}});